test/client/fixtures/sandbox/event/focus-blur-change-test.js

```diff
@@ -1042,3 +1042,26 @@ asyncTest('focus() must not raise the event if the element is in an invisible if
 
     document.body.appendChild(iframe);
 });
+
+test('should correctly handle the case when document.activeElement is null or an empty object (GH-768)', function () {
```

Comment:
I've tried to run this test locally in IE11 without your changes in src/client/sandbox/event/focus-blur.js and it's ok. Please ensure that this test is failed without your fix.